pure.zsh

```diff
@@ -543,10 +543,10 @@ prompt_pure_state_setup() {
 	fi
 
 	# show username@host if logged in through SSH
-	[[ -n $ssh_connection ]] && username='%F{242}%n@%m%f'
+	[[ -n $ssh_connection ]] && username='%F{${prompt_pure_colors[user]}}%n@%m%f'
 
 	# show username@host if root, with username in white
-	[[ $UID -eq 0 ]] && username='%F{white}%n%f%F{242}@%m%f'
+	[[ $UID -eq 0 ]] && username='%F{${prompt_pure_colors[root]}}%n%f%F{242}@%m%f'
```

Comment:
Missed a color here: `%F{242}`. Suggested: `%F{${prompt_pure_colors[host]}}` and also add above (in ssh section).

Reply:
Also intentional because I didn't grasp the full intent of the authors. Blaming those lines (https://github.com/sindresorhus/pure/pull/94/files & https://github.com/sindresorhus/pure/pull/108/files) it seems the formatting was an after thought. I went with your suggestion.